ibm_catalog.json

```diff
@@ -0,0 +1,501 @@
+{
+	"products": [
+		{
+			"label": "Apptio Cloudability",
+			"name": "apptio-cloudability-onboarding",
+			"version": "v0.0.1",
+			"product_kind": "solution",
+			"tags": [
+				"ibm_created",
+				"ibm_beta",
+                "integration"
+			],
+			"keywords": [
+				"Billing",
+				"Apptio",
+				"Cloudability",
+				"FinOps",
+				"Cloud Spend"
+			],
+			"short_description": "Add your IBM Cloud Account to Apptio Cloudability and start tracking your cloud spend.",
+			"long_description": "Simplify your IBM Cloud Account onboarding to Apptio Cloudability with this deployable architecture. Only an IBM Cloud IAM Api Key and Cloudability API key are required. This deployable architecture will take care of all of the necessary steps to get your IBM Cloud billing data into Apptio Cloudability. This includes the following:\n\n1. Creating Key Protect Encrypted COS Buckets to store your daily billing snapshots.\n2. Setup writing of billing snapshots to the COS Bucket.\n3. Grant Apptio Cloudability access to read from the COS buckets\n4. Add your IBM Cloud account to Cloudability and run access verification from Cloudability.\n\nYour IBM Cloud billing data will begin showing up in Apptio cloudability the next day. \nTo obtain your Apptio Cloudability API key visit https://app.apptio.com/cloudability#/settings/preferences under the section **Cloudability API** select **Enable API** which will generate an api key. If you don't have a Cloudability account start your tree trial now: https://www.apptio.com/products/cloudability/#cldy-trial.",
+			"provider_name": "apptio-cloudability",
```

Comment:
I think this only supports some predefined values. You going to want to go with "IBM"